docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -320,6 +320,53 @@ will be lost.
 $ kubectl delete -f alluxio-journal-volume.yaml
 ```
 
+#### Upgrade Alluxio
```

Comment:
We should have another section for "Updating Configuration"; can do that in a separate PR

Reply:
I'll do it in a separate PR. There's sth about live update configmap I wanna test and add too.